R-packages/modeltools/DESCRIPTION

```diff
@@ -0,0 +1,26 @@
+Package: modeltools
+Type: Package
+Title: Tools for Building COVID Forecasters and Related Models 
+Version: 0.1.0
+Authors@R:
+	c(
+    person(given = "Ryan",
+           family = "Tibshirani",
+           role = "aut"),
+    person(given = "Not Ryan",
+           family = "Tibshirani",
+           role = c("aut", "cre"),
+					 email = "not_ryantibs@cmu.edu"))
+Description: Tools for building COVID-19 models, mainly forecasting and hotspot
+    prediction models. This package is meant to be used in conjuction with the
+    covidcast and evalcast packages.
+Encoding: UTF-8
+License: MIT
+Imports: 
+    covidcast,
+    dplyr,
+		evalcast,
+    genlasso,
+    slider,
+    tidyr
```

Comment:
```suggestion
    tidyr
Remotes:
    local::../covidcast,
    local::../evalcast
```

Automatic dependency resolution (if installing `modeltools` without, e.g., `evalcast` already installed (cf. https://cran.r-project.org/web/packages/remotes/vignettes/dependencies.html).

Reply:
Thanks!  Good catch.  I'll stick to the fix used in the evalcast DESCRIPTION to be consistent (which is very similar and likely equivalent).